### PR TITLE
Adding story line about a drawn chocolate door

### PR DESCRIPTION
**Pull request by GranJDev**
This story ine is in the french folder.

Please review this @casey-collab, (Should have been done, so that there is no conflict)

### CODE REVIEW

File: french/dessine/porte.md [hunk 1]
```diff
@@ -3,5 +3,6 @@ La porte qui m'a attire le plus est la porte dessinee avec du miel. Elle brillai
 Donc je decide de
 
 [Manger un peu de miel](miel/manger-du-miel.md)
+[Essayer d'ouvrir la porte faite de chocolat](chocolat/essayer-ouvrir-chocolat.md)
```
Comment (casey-collab) [inline french/dessine/porte.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by pmallory** [comment]
@casey-collab can you review this PR please?

File: french/dessine/porte.md (hunk 1 above)
Comment (casey-collab) [inline french/dessine/porte.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**